src/foam/dao/AbstractDAO.js

```diff
@@ -284,6 +284,12 @@ foam.CLASS({
   ],
 
   methods: [
+    function find(key) {
```

Comment:
I think that would make sense, but I don't have time to implement it for `FilteredDAO` + all our network DAOs right now. Maybe file an issue?